### DISCUSSION TITLE
PWA functionality

### DISCUSSION THREAD

**Discussion by tintin10q** [srid/emanote#266] [Ideas]
Maybe this is already possible but with some simple additions the wiki could be made into a progressive web app. Just a manifest.json in the root. Is it already possible to make a PWA out of the wiki? Is this possible with neuron?

**Comment by srid** [comment]
With Emanote you can put `manifest.json` just at the top-level of your notebook directory, and it will be served as `/manifest.json`. For PWA, you might want to inject some meta vars in `<head>` as well, which can be done in the [`page.headHtml` section](https://github.com/srid/emanote/blob/a1348f07d3cb76766bd61f60c9e3829b5caecf37/default/index.yaml#L71-L72) of your index.yaml file.

We can also provide a default `manifest.json` in the default layer so that Emanote serves a PWA app by default.

> Is this possible with neuron?

With neuron, you have to put it inside `./static` and then add a tag to `<head>` specifying the path to the new manifest. I've done this, and it works.

**Comment by srid** [comment]
Related: https://github.com/srid/emanote/issues/175
